drafts/_sources/lab02-02.rst

```diff
@@ -34,27 +34,29 @@ Your program will be handling the following rules to pluralize nouns:
     | All other cases    | Add "s" to the end             | cat, cats         |
     +--------------------+--------------------------------+-------------------+
 
+Note that in order to handle cases where the word ends in "y" correctly, you will need to take some care. It is important that you order your conditions so that your code will check for the special case endings of "ay", "oy", "ey", and "uy" before simply checking whether a word ends in "y".
+
 You should name your file ``FILN_pluralizer.py``, where FILN is your first initial and last name, all lowercase, no space.
 
 Testing Your Program
 --------------------
 
-Although it is a good idea to come up with some of your own test cases to realize the limitations our own programs the following are some inputs that can also be used to verify that your program works as intended.  I would also recommend using the examples in the table above.
+Although it is a good idea to come up with some of your own test cases to realize the limitations your own code for this lab (and other labs) the following are some inputs you can use to verify that your program works as intended.  I would also recommend using the examples in the table above.
 
     +-----------+----------------+-------------------+
     | Input (#) | Input (word)   | Result            |
     +-----------+----------------+-------------------+
-    | 0         | word           | 0 words           |
+    | 0         | dog            | 0 dogs            |
     +-----------+----------------+-------------------+
     | 1         | city           | 1 city            |
     +-----------+----------------+-------------------+
     | 5         | fish           | 5 fishes          |
     +-----------+----------------+-------------------+
     | 33        | fly            | 33 flies          |
     +-----------+----------------+-------------------+
-    | 8         | wife           | 8 wives           |
+    | 8         | life           | 8 lives           |
```

Comment:
Political correctness encourages one to stay away from such examples. :-) and :-(